metricbeat/module/jolokia/jmx/data.go

```diff
@@ -33,9 +34,10 @@ const (
 
 type Entry struct {
 	Request struct {
-		Mbean string `json:"mbean"`
+		Mbean     string      `json:"mbean"`
+		Attribute interface{} `json:"attribute"`
```

Comment:
Can the attribute be of any type? I think it is safe to expect that it can only be `string`. Find the protocol here https://jolokia.org/reference/html/protocol.html

Reply:
Attribute here can be string or []interface {} for example, `v.Request.Attribute =  [HeapMemoryUsage NonHeapMemoryUsage]`

Reply:
Ok, I see, unfortunate :slightly_frowning_face: